kit/Watermark.hpp

```diff
@@ -99,24 +97,27 @@ class Watermark final
     /// Create bitmap that we later use as the watermark for every tile.
     const std::vector<unsigned char>* getPixmap(int width, int height)
     {
-        if (!_pixmap.empty() && width == _width && height == _height)
-            return &_pixmap;
-
-        _pixmap.clear();
-
-        _width = width;
-        _height = height;
-
         if (_loKitDoc == nullptr)
         {
             return nullptr;
         }
 
+        if (_pixmaps.find(width * height) != _pixmaps.end())
+        {
+            return &_pixmaps[width * height];
```

Comment:
I guess this is a 32bit key; if so I'd be inclined to have a cleanup that does width + height * 10000 - or somesuch =) but of course it's rather unlikely that we get problems here, unless there are duplicate portrait and landscape versions of a tile =)

Reply:
oh didnt think of that :) sure changing it now